env.template

```diff
@@ -86,8 +87,12 @@ SSL_CA_PATH=scos_test_ca.crt
 SSL_CERT_PATH=sensor01.pem
 SSL_KEY_PATH=sensor01.pem
 
-USB_DEVICE=Tektronix
-
+# Calibration action selection
+#    The action specified here will be used to attempt an onboard
+#    sensor calibration on startup, if no onboard calibration data
+#    is available on startup. The specified action must be available.
+CALIBRATE_ON_STARTUP=true
```

Comment:
Suggest removing CALIBRATE_ON_STARTUP as it is not used or modify the code to use it if there is a need to control this behavior.

Reply:
Removed in fc8335b77ac1f6d1a9ba480ef33dba3642a75f40